src/Test/Chai.purs

```diff
@@ -73,6 +73,13 @@ toInclude                = bindExpectation "to.include"
 toNotInclude             :: Expectation
 toNotInclude             = bindExpectation "to.not.include"
 
+toBeOk                   :: forall e. Expect -> Eff (chai :: Chai|e) Unit
+toBeOk                  x  = unsafeForeignProcedure ["expect", ""] $ "expect.to.be.ok"
+
+
+toNotBeOk                :: forall e. Expect -> Eff (chai :: Chai|e) Unit
+toNotBeOk               x  = unsafeForeignProcedure ["expect", ""] $ "expect.to.not.be.ok"
```

Comment:
So the expected use would look like this?

```haskell
expect true `toBeOk` -- this wont work because there is no second argument to `toBeOk`
toBeOk $ expect true -- so it has to be like this which is not intuitive
```

I would prefer usage look like this, also to document that `.to.be.ok` in chai is a truthy check

```haskell
expectToBeTruthy true
```